### ISSUE TITLE
[BUG] TypeError: sequence item 0: expected str instance, NoneType found

### ISSUE THREAD

**Issue by kota-tozawa** [xenova/chat-downloader#209]
## Basic information
- Program version: 0.2.5
- Python version: 3.10.4
- Operating system: Mac OS Venture Version 13.0.1

## Describe the bug
When trying to download a twitch chat, `TypeError: sequence item 0: expected str instance, NoneType found` occurs.

### If running from the command line, provide the following:
1. The command used (including the verbose tag, `-v`):
```
chat_downloader "https://www.twitch.tv/videos/1822957278" -v --output "./output/1822957278.json"
```
2. Output from the above command:
```
1:54:58 | (cheer 10000) まーー111: 空飛ぶやつなかったっけ？w
1:54:59 | (Prime Gaming) せとぎわ: 自殺願望者たち
1:55:03 | (2-Month Subscriber, cheer 10000) もの_00: この辺も戦場っぽくて草
1:55:03 | (2-Month Subscriber, Bits Leader 3) ようみゅ: みんな死んでくのおもろい
1:55:06 | (2-Month Subscriber, Bits Leader 1) るりえ: 前のやつに習ってどんどん落ちるｗ
[DEBUG] Session closed.
Traceback (most recent call last):
  File "/Users/xxxxxxx/.pyenv/versions/3.10.4/bin/chat_downloader", line 8, in <module>
    sys.exit(main())
  File "/Users/xxxxxxx/.pyenv/versions/3.10.4/lib/python3.10/site-packages/chat_downloader/cli.py", line 194, in main
    run(**args.__dict__)
  File "/Users/xxxxxxx/.pyenv/versions/3.10.4/lib/python3.10/site-packages/chat_downloader/chat_downloader.py", line 361, in run
    callback(message)
  File "/Users/xxxxxxx/.pyenv/versions/3.10.4/lib/python3.10/site-packages/chat_downloader/chat_downloader.py", line 358, in callback
    chat.print_formatted(item)
  File "/Users/xxxxxxx/.pyenv/versions/3.10.4/lib/python3.10/site-packages/chat_downloader/sites/common.py", line 304, in print_formatted
    safe_print(self.format(item), flush=flush)
  File "/Users/xxxxxxx/.pyenv/versions/3.10.4/lib/python3.10/site-packages/chat_downloader/chat_downloader.py", line 260, in <lambda>
    chat.format = lambda x: formatter.format(
  File "/Users/xxxxxxx/.pyenv/versions/3.10.4/lib/python3.10/site-packages/chat_downloader/formatting/format.py", line 166, in format
    substitution = re.sub(self._INDEX_REGEX, lambda match: self._replace(
  File "/Users/xxxxxxx/.pyenv/versions/3.10.4/lib/python3.10/re.py", line 209, in sub
    return _compile(pattern, flags).sub(repl, string, count)
  File "/Users/xxxxxxx/.pyenv/versions/3.10.4/lib/python3.10/site-packages/chat_downloader/formatting/format.py", line 166, in <lambda>
    substitution = re.sub(self._INDEX_REGEX, lambda match: self._replace(
  File "/Users/xxxxxxx/.pyenv/versions/3.10.4/lib/python3.10/site-packages/chat_downloader/formatting/format.py", line 96, in _replace
    value = separator.join(
TypeError: sequence item 0: expected str instance, NoneType found
```
If the output is too long, you can attach a text file or remove output which does not constitute to the problem.

### Otherwise, if using the python module, provide the following:

1. A [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example):
```python
# python code
```
2. Output, traceback or other information relating to the bug:
```
[output or description]
```

## Additional context/information
Add any other context or information about the problem here.

**Comment by xenova** [comment]
Thanks for the report. Fixing now.

**Comment by kota-tozawa** [comment]
Thank you for dealing with it so quickly.

**Comment by xenova** [comment]
No worries :) Turns out you found a bug with twitch:

```json
"userBadges":[
   {
      "id":"Ozs=",
      "setID":"",
      "version":"",
      "__typename":"Badge"
   },
   {
      "id":"Yml0czsxMDAwOw==",
      "setID":"bits",
      "version":"1000",
      "__typename":"Badge"
   }
],
```

is what is returned by their API. As you can see, the first badge is missing a version and an ID.

**Comment by kota-tozawa** [comment]
Oh, I understand. Thank you again for your swift investigation 😀

**Comment by xenova** [comment]
Fix is live in version 0.2.6 :) 

```bash
pip install --upgrade chat-downloader
```
